athena-oracle/src/main/java/com/amazonaws/athena/connectors/oracle/OracleMetadataHandler.java

```diff
@@ -98,7 +95,7 @@ public OracleMetadataHandler()
      */
     public OracleMetadataHandler(final DatabaseConnectionConfig databaseConnectionConfig)
     {
-        this(databaseConnectionConfig, new OracleJdbcConnectionFactory(databaseConnectionConfig, JDBC_PROPERTIES, new DatabaseConnectionInfo(OracleConstants.ORACLE_DRIVER_CLASS, OracleConstants.ORACLE_DEFAULT_PORT)));
+        this(databaseConnectionConfig, new OracleJdbcConnectionFactory(databaseConnectionConfig, null, new DatabaseConnectionInfo(OracleConstants.ORACLE_DRIVER_CLASS, OracleConstants.ORACLE_DEFAULT_PORT)));
```

Comment:
if this param is always null now, can we just get rid of the param from the `OracleJdbcConnectionFactory` constructor?